apps/api/src/app/notification-template/usecases/update-notification-template/update-notification-template.usecase.ts

```diff
@@ -16,14 +16,17 @@ import { UpdateMessageTemplateCommand } from '../../../message-template/usecases
 import { UpdateMessageTemplate } from '../../../message-template/usecases/update-message-template/update-message-template.usecase';
 import { CreateChange } from '../../../change/usecases/create-change.usecase';
 import { CreateChangeCommand } from '../../../change/usecases/create-change.command';
+import { mongo } from 'mongoose';
```

Comment:
Can we remove this one? I think it's better to not import mongoose directly within the API and only access it from the DAL lib